usng4j-api/src/main/java/org/codice/usng4j/UtmCoordinate.java

```diff
@@ -63,6 +63,12 @@
      */
     Character getLattitudeBand();
 
+    /**
+     *
+     * @return the N/S indicator if using northing values with hemisphere indicator
+     */
+    Character getNSIndicator();
```

Comment:
✏️ We should probably have constants somewhere to define what values this character can be.

Reply:
Or use an enum...

Reply:
I sort of want to get rid of the public constructors and move to factory methods with an enum, but that felt a bit much for this PR.

Reply:
But maybe I'll just tackle that now.